src/test/regress/citus_tests/run_test.py

```diff
@@ -81,6 +81,8 @@
     test_schedule = 'base_isolation_schedule'
 elif "failure" in test_schedule:
     test_schedule = 'failure_base_schedule'
+elif "enterprise" in test_schedule:
+    test_schedule = 'enterprise_minimal_schedule'
```

Comment:
this is a new schedule created only for the flaky check. Because neither minimal nor base schedules is sufficient for the test files in the enterprise schedule. Mainly because of the `multi_data_types`

Reply:
makes sense, we should maybe get rid of enterprise schedule at some point